Homework_9/file_txt.py

```diff
@@ -0,0 +1,15 @@
+# 1. Вам потрібно створити текстовий файл(.txt) з текстом і написати програму, яка зкопіює дані з вашого файлу в інший текстовий файл
+
+my_str = """
+Some information for testing work with txt-files.
+Another line with some words here!
+"""
+
+with open('first.txt', 'w') as first_file:
+    first_file.write(my_str)
+
+with open('first.txt', 'r') as first_file:
+    data = first_file.read()
+
+with open('second.txt', 'w') as second_file:
+    second_file.write(data)
```

Comment:
Can be also
![image](https://github.com/SerhiiKotsar/hillel_python_basic/assets/51053483/c1767724-67d7-47ae-8490-492f29e557fa)

Reply:
it does not work on my PyCharm